agent/util-scripts/pbench-tool-meister

```diff
@@ -363,7 +363,8 @@ class ToolMeister(object):
 
     def __init__(self, pbench_bin, params, redis_server, logger):
         self.logger = logger
-        self.persist_tools = ["node-exporter", "dcgm"]
+        self.tool_metadata = metaclass.ToolMetadata(self.logger, redis_server=redis_server)
+        self.persist_tools = list(self.tool_metadata.getPersistentTools().keys())
```

Comment:
Why not have `getPersistentTools()` return a `list`?  Seems like too much knowledge of the implementation leaking out here.